### ISSUE TITLE
Questions about the choice of backbone

### ISSUE THREAD

**Issue by Rbrq03** [Atten4Vis/LW-DETR#28]
Hey there,
  Congrats to the great and elegant work LW-DETR. 
  I have a question about the choice of backbone, why you choose ViT instead of some more efficient backbone like SwinT, and have you ever have some abl experiments on the backbone?

**Comment by bartbuaa** [comment]
Trying to answer this question for the author. 
The paper mentions that the backbone network is pre-trained using a self-supervised method (MIM) on the OBJ365 dataset. The Vision Transformer (ViT) patches the images, making it easier to employ mask learning. Additionally, the ViT used in the paper has been modified. Some layers employ local self-attention within patches, while other layers apply the  global self-attention on the whole feature map in the normal way with higher complexity. This is analogous to the window/shifted window self-attention proposed by Swin. Besides, this method was first proposed by VitDet. However, the performance of LW-DETR has not been compared with the DETR families, possibly because this model tends to prioritize speed.

**Comment by Rbrq03** [comment]
@bartbuaa Thanks for your comprehensive answer!
